rules/windows/sysmon/silenttrinity_stager_msbuild_activity.yml

```diff
@@ -0,0 +1,26 @@
+title: Silenttrinity Stager Msbuild Activity
+id: 50e54b8d-ad73-43f8-96a1-5191685b17a4
+description: Detects a possible remote connections to Silenttrinity c2
+references:
+    - https://www.blackhillsinfosec.com/my-first-joyride-with-silenttrinity/
+tags:
+    - T1127.001
+    - TA0002
+    - T1127
+    - attack.defense_evasion
+status: experimental
+author: Kiran kumar s, oscd.community
+date: 2020/10/11
+logsource:
+    category: sysmon
```

Comment:
Please change to:
`category: network_connection`

Reply:
Resloved

Reply:
can anyone help why the checks are failing ?

Reply:
Hello @skirankumar! Most probably it's just broken. Most of the contributors are having pretty much the same issues with it. So don't worry about it for now (:

Reply:
Okay

Reply:
Most probably event Id 3 is required to narrow down the logs

Reply:
@skirankumar it is already covered by the `network_connection` category, no need to use event id here (: